source/win-main/Editor.vue

```diff
@@ -591,22 +591,54 @@ export default {
      */
     editorMousedown (event) {
       if (event.target !== this.$refs.editor) {
-        // return // Only handle if the event's target is the editor itself
+        return 
+      }
+      // if the button is the mouse left button, start selecting lines
+      if(event.button === 0){
```

Comment:
You can merge this if-statement with the one above (or do a check for event.button !== 0 and return) — that would improve readability and reduce the chance that any changes afterwards end up accidentally AFTER the closing bracket!